app/notification.go

```diff
@@ -657,6 +753,44 @@ func (a *App) getMentionKeywordsInChannel(profiles map[string]*model.User, allow
 	return keywords
 }
 
+// insertGroupMentions adds group members in the channel to Mentions, adds group members not in the channel to OtherPotentialMentions
+// returns false if no group members present in the team that the channel belongs to
+func (a *App) insertGroupMentions(group *model.Group, channel *model.Channel, profileMap map[string]*model.User, mentions *ExplicitMentions) (bool, *model.AppError) {
+	userMentioned := false
+	groupMembers, err := a.Srv().Store.Group().GetMemberUsers(group.Id)
+	if err != nil {
+		return userMentioned, err
+	}
+
+	if mentions.Mentions == nil {
+		mentions.Mentions = make(map[string]MentionType)
```

Comment:
As discussed with @catalintomai this will be replaced with a new store function for `GetMemberUsersInTeam` saving us from having to do the call to `.GetMemberUsersNotInChannel` however I will still commit that store function since it will be useful when we need to do group invites to channel